src/pow/pow.cpp

```diff
@@ -31,14 +31,10 @@ uint32_t GetNextWorkRequired(const CBlockIndex *pindexPrev,
         return pindexPrev->nBits;
     }
 
-    if (IsAxionEnabled(params, pindexPrev)) {
+    if (pindexPrev->pprev) {
         return GetNextASERTWorkRequired(pindexPrev, pblock, params);
     }
 
-    if (IsDAAEnabled(params, pindexPrev)) {
-        return GetNextDAAWorkRequired(pindexPrev, pblock, params);
-    }
-
     return GetNextEDAWorkRequired(pindexPrev, pblock, params);
```

Comment:
This is used for the Genesis block and nothing else? Something seems fishy here.

Reply:
`GetNextEDAWorkRequired` is just used for the block after genesis.

ASERT requires two existing blocks.

I decided it's easiest to keep the old code for this one block; we can add a special case to GetNextASERTWorkRequired later.

Or what do you have in mind?